index/embedding.py

```diff
@@ -28,19 +27,42 @@ def get_embedding(self, text: str, model="text-embedding-ada-002"):
                 return None
             if isinstance(text, str):
                 text = text.replace("\n", " ")
-            return openai.Embedding.create(input=[text], model=model)['data'][0]['embedding']
+            return openai.Embedding.create(input=[text], model=model)["data"][0][
```

Comment:
Not really a problem syntax wise but I am wondering about this formatting choice

Reply:
I'm using Black Formatter on VScode and it automatically formats on save. I will try to disable it.